agent/lm_agent/backend_utils.py

```diff
@@ -54,9 +55,9 @@ def _write_token_to_cache(token: str):
 
     token_path = settings.CACHE_DIR / TOKEN_FILE_NAME
     try:
-        token_path.touch(mode=0o600, exist_ok=True)
+        token_path.touch(mode=0o660, exist_ok=True)
```

Comment:
As we discussed in Slack, I think we will need to keep separate files for the different users. We need to make sure that token files are stored with at least 0o600 permissions.

Additionally, I suggest using `chmod` after the `write_text()` call every time instead of using `touch()`.

Reply:
I updated the code to create one file for each user, with permission 600.